pkg/templates/node_1.19.go

```diff
@@ -43,6 +43,9 @@ systemd:
           contents: |
             [Service]
             Environment="DOCKER_OPTS=--log-opt max-size=5m --log-opt max-file=5 --ip-masq=false --iptables=false --bridge=none"
+            {{- if .CGroupsV1 }}
+            Environment="DOCKER_CGROUPS=--exec-opt native.cgroupdriver=cgroupfs"
```

Comment:
this line should be `Environment="DOCKER_CGROUPS=--exec-opt native.cgroupdriver=systemd"`
cc @SchwarzM